src/couch/src/couch_bt_engine.erl

```diff
@@ -752,6 +848,16 @@ copy_security(#st{header = Header} = St, SecProps) ->
         needs_commit = true
     }}.
 
+copy_props(#st{header = Header} = St, Props) ->
```

Comment:
We are not really copying props here. So the name of the function is not accurate. Should we call it `write_props` instead?